rllib/models/modelv2.py

```diff
@@ -333,71 +333,6 @@ def is_time_major(self) -> bool:
         """
         return self.time_major is True
 
-    # TODO: (sven) Experimental method.
```

Comment:
Was moved to SampleBatch instead (makes more sense there).
SampleBatch converts itself into an single-timestep input-dict (which is also a SampleBatch).